lib/models/addon.js

```diff
@@ -730,7 +777,7 @@ var Addon = CoreObject.extend({
     if (addonTemplates) {
       standardTemplates = new Funnel(addonTemplates, {
         srcDir: '/',
-        destDir: 'modules/' + this.name + '/templates'
+        destDir: this.name + '/templates'
```

Comment:
@stefanpenner - The `modules/` funneling is done a tad bit later. In this case, we need the paths on disk to match what we expect the named AMD moduleIds will be.